Sources/Support/DebugUI/DebugContentViews.swift

```diff
@@ -414,6 +435,78 @@ private struct DebugPackageView: View {
+                switch error {
+                case let .invalidProducts(products):
+                    Section("Product Validation Issues") {
+                        ForEach(products, id: \.identifier) { product in
+                            if product.status != .valid {
+                                VStack(alignment: .leading, spacing: 8) {
+                                    HStack(alignment: .center, spacing: 8) {
+                                        product.status.icon.imageScale(.small)
+                                        Text(product.identifier)
+                                    }
+                                    Text(product.statusMessage)
+                                        .foregroundStyle(.secondary)
+                                        .font(.caption)
+                                }
+                            }
+                        }
+                    }
+                case .noOfferings:
+                    Section("Offering Validation Issues") {
+                        VStack(alignment: .leading, spacing: 8) {
+                            HStack(alignment: .center, spacing: 8) {
+                                Image(systemName: "exclamationmark.triangle.fill")
+                                    .foregroundColor(.yellow)
+                                Text("No offerings configured")
+                            }
+                            Text(
+                                """
+                                While offerings are not mandatory, they are the way to 'offer' products \
+                                to your customers on your paywalls.
+                                """
+                            )
+                            .font(.caption)
+                            .foregroundStyle(.secondary)
+                        }
+                    }
+                case let .offeringConfiguration(offerings):
+                    Section("Offering Validation Issues") {
+                        ForEach(offerings, id: \.identifier) { offering in
+                            if offering.status != .passed {
+                                VStack(alignment: .leading, spacing: 8) {
+                                    HStack(alignment: .center, spacing: 8) {
+                                        offering.status.icon.imageScale(.small)
+                                        Text(offering.identifier)
+                                    }
+                                    Text(
+                                        offering.packages.isEmpty ?
+                                        "Offerings must have at least one package" :
+                                        """
+                                        Products \(offering.packages.map { "'\($0.productIdentifier)'" }.formatted()) \
+                                        in the offering have issues, check the products list above for more details.
+                                        """
```

Comment:
this is unreasonably exciting to me

Reply:
Honestly! ❤️ I can't wait to ship thisssss 🚀💪